client/components/data/query-rewind-status/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
```

Comment:
I see a lint error here for using `React.PropTypes` instead of standalone `prop-types` module.